notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<p>* Really great refactor! Super clear now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Thank you!</p>